sdk/node/Libplanet.Node.Executable/appsettings.Development.json

```diff
@@ -12,7 +12,8 @@
     }
   },
   "Swarm": {
-    "IsEnabled": true
+    "IsEnabled": true,
+    "AppProtocolVersion": "200210/AB2da648b9154F2cCcAFBD85e0Bc3d51f97330Fc/MEUCIQCBr..8VdITFe9nMTobl4akFid.s8G2zy2pBidAyRXSeAIgER77qX+eywjgyth6QYi7rQw5nK3KXO6cQ6ngUh.CyfU=/ZHU5OnRpbWVzdGFtcHUxMDoyMDI0LTA3LTMwZQ=="
```

Comment:
We do not need this. This APV is just for NineChronicles, not libplanet. Could you make a new APV and use it?

Reply:
😳

Reply:
Plus, not only need APV but also a Trusted App Protocol Version Signer. If do not validate APV is signed by the correct private key, anyone can be spamming with the highest APV and sabotage network.

Reply:
AppProtocolVersion in appsettings.json was deleted. This value does not need to be here.